app/components/homepage/homeservices.module.css

```diff
@@ -0,0 +1,31 @@
+.cardHomeServices {
+  overflow: hidden;
+  background: white;
+  box-shadow: 0 0 15px rgba(0, 0, 0, 0.2);
+}
+
+.cardHomeServices > img {
```

Comment:
is there a good reason for ussing selectors here?

i think its a better idea to have classes instead of selectors

sometimes its better to duplicate code a bit if it makes it simple to understand

Reply:
you are right, thank you for the correction. I will delete it.